### PR TITLE
Add permission.xml files to appropriate mpConfig FAT apps

### PR DESCRIPTION
**Pull request by Joseph-Cass**
Resolves defects 279205 and 279206

### CODE REVIEW

**Comment by Joseph-Cass** [comment]
#build

File: dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/LibertySpecificConfigTests.java [hunk 1]
```diff
@@ -106,6 +112,9 @@ public static void setUp() throws Exception {
     @AfterClass
     public static void tearDown() throws Exception {
         server.stopServer("CWWKE0921W", "CWWKE0912W");
+        //CWWKE0912W: Current Java 2 Security policy reported a potential violation of Java 2 Security Permission.
+        //CWWKE0921W: Current Java 2 Security policy reported a potential violation of Java 2 Security Permission. The application needs to have permissions addedPermission:
+        //("com.ibm.oti.shared.SharedClassPermission" "com.ibm.ws.microprofile.appConfig.classLoaders.test.CustomClassLoaderError" "read")
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/LibertySpecificConfigTests.java:117]:
```suggestion
        // CWWKE0912W, CWWKE0921W: Ignore Java 2 Security policy warning about "com.ibm.oti.shared.SharedClassPermission"
        // We expect an access control exception here to be safely caught and handled, so we just need to ignore the warning generated by our FAT infrastructure when running locally.
```

**Comment by Joseph-Cass** [comment]
#build

**Review by Azquelt — APPROVED** [review]

**Review by Azquelt — APPROVED** [review]

**Comment by Joseph-Cass** [comment]
#run-libby-bot

**Pull request merged**